### PR TITLE
WL-2951 fix

### PR DESCRIPTION
**Pull request by ghost**
Includes the code from [reference pull request 1](https://github.com/ox-it/wl-reference/pull/1).

### CODE REVIEW

**Comment by buckett** [comment]
You need to either rebase or merge sakai-2.8.x into this branch.

**Comment by ghost** [comment]
Merged

File: library/src/webapp/js/headscripts.js [hunk 1]
```diff
@@ -666,4 +668,13 @@ function forceLinksInNewWindow() {
             }
         }
     });
+}
+
+// rewrites old weblearn links to the new address, called from forceLinksInNewWindow(),
+// this could have its own entry in sakai.properties.
+function rewriteWebLearnHref(link) {
+    if(link.href.match("^http://weblearn.ox.ac.uk|^http://beta.weblearn.ox.ac.uk")) {
```
Comment (buckett) [inline library/src/webapp/js/headscripts.js]: Missing the check to see that link.href is defined.

**Comment by ghost** [comment]
Added href check.

**Pull request merged**